pkg/migration/controllers/migration.go

```diff
@@ -1499,10 +1513,120 @@ func (m *MigrationController) getParsedLabels(
 	return m.getParsedMap(labels, clusterPair)
 }
 
+// updateOwnerReferenceOnPVC updates owner reference on PVC objects
+// on destination cluster
+func (m *MigrationController) updateOwnerReferenceOnPVC(
+	migration *stork_api.Migration,
+	objects []runtime.Unstructured,
+	clusterPair *stork_api.ClusterPair,
+	pvcsWithOwnerRef []v1.PersistentVolumeClaim,
+) error {
+	remoteConfig, err := getClusterPairSchedulerConfig(migration.Spec.ClusterPair, migration.Namespace)
+	if err != nil {
+		return err
+	}
+	remoteAdminConfig := remoteConfig
+	// Use the admin cluter pair for cluster scoped resources if it has been configured
+	if migration.Spec.AdminClusterPair != "" {
+		remoteAdminConfig, err = getClusterPairSchedulerConfig(migration.Spec.AdminClusterPair, m.migrationAdminNamespace)
+		if err != nil {
+			return err
+		}
+	}
+	adminClient, err := kubernetes.NewForConfig(remoteAdminConfig)
+	if err != nil {
+		return err
+	}
+	remoteInterface, err := dynamic.NewForConfig(remoteConfig)
+	if err != nil {
+		return err
+	}
+	remoteAdminInterface := remoteInterface
+	if migration.Spec.AdminClusterPair != "" {
+		remoteAdminInterface, err = dynamic.NewForConfig(remoteAdminConfig)
+		if err != nil {
+			return err
+		}
+	}
+
```

Comment:
nit: Lets move this into a separate function and pass all the objects to this function and applyResources

Reply:
Moved this to a function and returning a struct of all the params. We can extract the respective param from the struct.